owiddata/owiddata-stats.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@RLordan this is what might interest you -- the current data we're pulling from OWID.

Reply:
Super cool @rando2 , I am looking forward to this!